spec/features/cdr/cdr_history/cdr_filter_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe 'Cdrs index page', type: :feature, js: true do
+  subject do
+    within_filters { click_submit('Filter') }
+  end
+
+  include_context :login_as_admin
+  let!(:cdrs) { create_list(:cdr, 2) }
+
+  before do
+    Cdr::Cdr.delete_all
+    visit cdrs_path
+  end
+
+  describe 'filtering', js: true do
+    context 'with filter by Tagged' do
+      # before do
+      #
+      # end
+
+      let!(:tag) { create :routing_tag }
+      let!(:cdr_tagged) { create :cdr, routing_tag_ids: [tag.id] }
+
+      it 'shows filtered records with routing_tags' do
+        within_filters do
+          select :Yes, from: :Tagged
+        end
```

Comment:
it would be nicer if you do filter selection in `before do ... end` and provide value as `let()`

```ruby
before do
   within_filters do
    fill_in_chosen 'Tagged', with: tagged_filter_value
  end
end
let(:tagged_filter_value) { 'Yes' }
```

also what is `select :Yes, from: :Tagged` ? is it working? I'm not familiar with that

Reply:
select :Yes, from: :Tagged it workin, saw in spec/features/routing/dialpeers/filters_spec.rb. But i will rewrite this

Reply:
fill_in_chosen method is not suitable for this check, used method select_by_value